common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -541,6 +544,51 @@ public Object run() {
                 DIRECT_BUFFER_CONSTRUCTOR != null ? "available" : "unavailable");
     }
 
+    private static Method getVirtualThreadCheckMethod() {
+        if (JAVA_VERSION < 19) {
+            return null;
+        }
+        try {
+            Method isVirtualMethod = Thread.class.getMethod("isVirtual");
```

Comment:
For this same change in Netty 4.2 we could use `MethodHandle::invokeExact`

Reply:
A quick local test shows performance improved after switched to `MethodHandle::invokeExact`.